README.md

```diff
@@ -108,6 +109,14 @@ system as a `bolt_controller` in Hiera.
 simp_bolt::bolt_controller: true
 ```
 
+To configure a Bolt controller to utilize a SIMP Omni-Environment, specify
+the name of the environment in Hiera.
+
+```yaml
+simp_bolt::simp_environment: true
+simp_bolt::simp_environment_name: bolt
+```
+
```

Comment:
A little misleading?  You still have to do the work to generate the omni environment yourself.   Perhaps, limiting the comment to 'To configure a Bolt controller to use a module path for a SIMP Omni-Environment'?

Reply:
Clarified that the environment needs to be created first.  We should be able to generate all three environments that compose the SIMP Omni-Environment.  By default, the Puppet environment and secondary environments are specified in the bolt.yaml file and the writable environment is in the cache directory (vardir), which Puppet/Bolt determine automatically.